tracee-rules/main.go

```diff
@@ -146,6 +146,10 @@ func main() {
 				Usage: "listening address of the pprof endpoints server",
 				Value: ":7777",
 			},
+			&cli.BoolFlag{
+				Name:  "enable-parsed-events",
```

Comment:
As mentioned in other PRs we should use common prefix for OPA / Rego related settings

Reply:
updated with `rego` prefix